pom.xml

```diff
@@ -484,6 +484,7 @@
     <version.org.jvnet.jaxb2.maven2.maven-jaxb2-plugin>0.14.0</version.org.jvnet.jaxb2.maven2.maven-jaxb2-plugin>
     <version.org.mock-server>5.11.1</version.org.mock-server>
     <version.org.apache.kafka>2.8.1</version.org.apache.kafka>
+    <version.shade.plugin>3.2.4</version.shade.plugin>
```

Comment:
@gorky `version.shade.plugin` is defined but it's not referenced in `<pluginManagement>`. Is it a mistake?

Reply:
@tkobayas it is not a mistake. Drools use jboss-parent as parent and plugins are declared there so shade plugin too https://github.com/jboss/jboss-parent-pom/blob/jboss-parent-36/pom.xml#L535-L539

Reply:
Oops, sorry. Thank you for pointing that out! @mareknovotny